CRT.c

```diff
@@ -987,15 +1048,14 @@ void CRT_handleSIGSEGV(int signal) {
       "============================\n"
       "Please check at https://htop.dev/issues whether this issue has already been reported.\n"
       "If no similar issue has been reported before, please create a new issue with the following information:\n"
-      "\n"
-      "- Your "PACKAGE" version ("PACKAGE" --version)\n"
-      "- Your OS and kernel version (uname -a)\n"
-      "- Your distribution and release (lsb_release -a)\n"
-      "- Likely steps to reproduce (How did it happened?)\n"
+      "  - Your "PACKAGE" version ("PACKAGE" --version)\n"
+      "  - Your OS and kernel version (uname -a)\n"
+      "  - Your distribution and release (lsb_release -a)\n"
+      "  - Likely steps to reproduce (How did it happened?)\n"
```

Comment:
There's a grammatical error there - @cgzones while here could you change "How did it happened?" to "How did it happen?"  Thanks!